spec/src/lib.rs

```diff
@@ -247,12 +256,6 @@ pub mod test {
         let chain_spec = dev.unwrap();
         let tx = chain_spec.build_system_cell_transaction().unwrap();
 
-        // Tx and Output hash will be used in some test cases directly, assert here for convenience
-        assert_eq!(
-            format!("{:x}", tx.hash()),
-            "48168c5b2460bfa698f60e67f08df5298b1d43b2da7939a219ffd863e1380d11"
-        );
-
```

Comment:
This test makes no sense.

Reply:
This is for developers who need to hard-coded `always_success` hash. However, it is going to be removed in #562 .